pkg/collector/corechecks/oracle-dbm/oracle.go

```diff
@@ -283,6 +295,16 @@ func (c *Check) Configure(integrationConfigDigest uint64, rawInstance integratio
 	c.tags = append(c.tags, fmt.Sprintf("dbms:%s", common.IntegrationName), fmt.Sprintf("ddagentversion:%s", c.agentVersion))
 
 	c.tagsString = strings.Join(c.tags, ",")
+
+	c.fqtCache = ttlcache.NewCache()
+	c.fqtCache.SetTTL(time.Duration(60 * time.Minute))
+	c.planCache = ttlcache.NewCache()
+	var planCacheRetention = c.config.QueryMetrics.PlanCacheRetention
+	if planCacheRetention == 0 {
+		planCacheRetention = 1
+	}
+	c.planCache.SetTTL(time.Duration(time.Duration(planCacheRetention) * time.Minute))
```

Comment:
I think you can skip one type conversion here:
```suggestion
	c.planCache.SetTTL(time.Duration(planCacheRetention) * time.Minute)
```